orttraining/orttraining/test/training_ops/cuda/softmax_test.cc

```diff
@@ -65,9 +70,7 @@ TEST(CudaKernelTest, SoftmaxGrad_SmallTensor) {
   TestSoftmaxGrad(dY_dims, Y_dims, dX_dims, per_sample_tolerance, relative_per_sample_tolerance);
 }
 
-// TODO fix flaky test
-// failing random seed: 552621640
-TEST(CudaKernelTest, DISABLED_SoftmaxGrad_LargeTensor) {
+TEST(CudaKernelTest, SoftmaxGrad_LargeTensor) {
```

Comment:
With the fix, do we still need to relax the relative_per_sample_tolerance to make this test pass?

Reply:
I have changed relative_per_sample tolerance from 5e-3 to 1e-4 to keep it in parity with PyTorch [test](https://github.com/pytorch/pytorch/blob/ee23944f46a89208ccba26c8fa0c97fe585265ac/caffe2/python/serialized_test/serialized_test_util.py#L251), by doing so we have tightened the relative_per_sample_tolerance and tests still pass when I run them in a loop for 1000 times (small tensor) and 100 times (large tensor).